src/main/java/com/squareup/protoparser/ProtoSchemaParser.java

```diff
@@ -700,6 +701,52 @@ private String readComment() {
     }
   }
 
+  private String tryAppendTrailingDocumentation(String documentation) {
+    // Search for a '/' character ignoring whitespace.
+    while (pos < data.length) {
+      char c = data[pos];
+      if (c == ' ' || c == '\t') {
+        pos++;
+      } else if (c == '/') {
+        pos++;
```

Comment:
I'll attempt this in a follow-up. Since multi-line comments are not officially supported (even in proto3, wtf?), I'm not going to lose sleep if we can't easily support it.